sklearn/cross_validation.py

```diff
@@ -323,10 +325,10 @@ class KFold(_BaseKFold):
     def __init__(self, n, n_folds=3, shuffle=False,
                  random_state=None):
         super(KFold, self).__init__(n, n_folds, shuffle, random_state)
-        self.idxs = np.arange(n)
+        self.mask_idxs = np.arange(n)
```

Comment:
Because it conflicts with a field with the same name used in another subclass of `_PartitionIterator`